### DISCUSSION TITLE
What is the GAL meaning?

### DISCUSSION THREAD

**Discussion by heitaoflower** [ezEngine/ezEngine#1007] [Q&A]
Hi,
I wonder what the abbreviation for GAL is here? Thanks.

such ezGALDevice.

**Comment by na2axl** [comment]
I think it means Graphic Abstraction Layer

**Reply by heitaoflower** [reply]
Thanks.

**Comment by jankrassnigg** [comment]
"Graphics Abstraction Layer". Others use "RHI" for, I think, "Render Hardware Interface" (not entirely certain). Basically it means that it is an abstraction for OpenGL / D3D / Vulkan.

**Reply by heitaoflower** [reply]
Got it, Thank you very much.
